samples/snippets/src/main/java/com/example/analytics/QuickstartSample.java

```diff
@@ -16,82 +16,74 @@
-import com.google.analytics.data.v1alpha.RunReportResponse;
+import com.google.analytics.data.v1beta.BetaAnalyticsDataClient;
+import com.google.analytics.data.v1beta.DateRange;
+import com.google.analytics.data.v1beta.Dimension;
+import com.google.analytics.data.v1beta.Metric;
+import com.google.analytics.data.v1beta.Row;
+import com.google.analytics.data.v1beta.RunReportRequest;
+import com.google.analytics.data.v1beta.RunReportResponse;
 
 public class QuickstartSample {
 
   // This is an example snippet that calls the Google Analytics Data API and runs a simple report
   // on the provided GA4 property id.
-  static void sampleRunReport(String ga4PropertyId) throws Exception {
-    // Instantiates a client using default credentials.
-    // See https://cloud.google.com/docs/authentication/production for more information
-    // about managing credentials.
-    try (AlphaAnalyticsDataClient analyticsData = AlphaAnalyticsDataClient.create()) {
+  static void sampleRunReport(String propertyId) throws Exception {
+    /**
+     * TODO(developer): Uncomment this variable and replace with your
+     * Google Analytics 4 property ID before running the sample.
+     */
+    // propertyId = "YOUR-GA4-PROPERTY-ID";
+
+    // [START google_analytics_data_initialize]
+    // Using a default constructor instructs the client to use the credentials
+    // specified in GOOGLE_APPLICATION_CREDENTIALS environment variable.
+    try (BetaAnalyticsDataClient analyticsData = BetaAnalyticsDataClient.create()) {
+      // [END google_analytics_data_initialize]
+
+      // [START google_analytics_data_run_report]
       RunReportRequest request = RunReportRequest.newBuilder()
-          .setEntity(Entity.newBuilder().setPropertyId(ga4PropertyId))
+          .setProperty("properties/" + propertyId)
           .addDimensions(
               Dimension.newBuilder().setName("city"))
           .addMetrics(Metric.newBuilder().setName("activeUsers"))
           .addDateRanges(
               DateRange.newBuilder().setStartDate("2020-03-31").setEndDate("today")).build();
 
-      // Make the request
+      // Make the request.
       RunReportResponse response = analyticsData.runReport(request);
+      // [END google_analytics_data_run_report]
 
+      // [START google_analytics_data_print_report]
       System.out.println("Report result:");
+      // Iterate through every row of the API response.
       for (Row row : response.getRowsList()) {
         System.out.printf("%s, %s%n", row.getDimensionValues(0).getValue(),
             row.getMetricValues(0).getValue());
       }
+      // [END google_analytics_data_print_report]
     }
   }
 
+
   public static void main(String... args) throws Exception {
-    // TODO(developer): Replace this variable with your GA4 property ID before running the sample.
-    String ga4PropertyId = "GA4 PROPERTY ID";
-    sampleRunReport(ga4PropertyId);
+    sampleRunReport("YOUR-GA4-PROPERTY-ID");
   }
```

Comment:
Again, you might wish to moe this above your class.

Reply:
Done.